include/JSystem/JParticle/JPAResource.h

```diff
@@ -66,7 +66,7 @@ class JPAResource {
     /* 0x34 */ JPAKeyBlock** mpKeyBlocks;
     /* 0x38 */ u16 const* mpTDB1;
     /* 0x3C */ u16 mUsrIdx;
-    /* 0x3E */ u8 mFieldBlockNum;
+    /* 0x3E */ u8 fieldBlockNum;
```

Comment:
Let's keep this unchanged?

Reply:
Ahh yes. Next time I’m at my computer I’ll revert that. I tried to be careful when doing that change but it  seems I overlooked those files